### ISSUE TITLE
canu: cant find trimmed reads in the directory

### ISSUE THREAD

**Issue by sunilthorat** [marbl/canu#2156]
I have installed canu v1.5 on conda environment since it was not executing though it said successfully installed in particular directory with all relevant files.
Following the Canu Tutorial at [](https://canu.readthedocs.io/en/latest/quick-start.html) For PacBio data I performed the below command:

```
**canu  -p ecoli -d ecoli-pacbio  genomeSize=4.8m  -pacbio-corrected pacbio.fastq**
```
I made corrections by adding "corrected" after -pacbio since there was an ERROR as below:
```
ERROR:  Invalid command line option '-pacbio'.  Did you forget quotes around options with spaces?
ERROR:  File supplied on command line; use -s, -pacbio-raw, -pacbio-corrected, -nanopore-raw, or -nanopore-corrected.
```
After sucessfully running the canu command, I got this error:

```
-- Finished stage 'obt-dumpReads', reset canuIteration.
-- Trimmed reads saved in 'trimming/ecoli.trimmedReads.fasta.gz'
================================================================================
Don't panic, but a mostly harmless error occurred and Canu stopped.
canu release v1.5 failed with:
  can't find trimmed reads 'ecoli.trimmedReads*' in directory '/home/server/ecoli-pacbio'
```
I checked the directory, there was not ecoli.trimmedReads however the files (ecoli.1.trimmedReads.*) were created at 
"~/ecoli-pacbio/trimming/3-overlapbasedtrimming" directory.

Could you please suggest me corrections so that I can run the Tutorial and understand the functioning of "camu" for my plant genome research.

**Comment by brianwalenz** [comment]
I'd suggest using the latest release (https://github.com/marbl/canu/releases/tag/v2.2) instead of the five years old v1.5.

**Comment by skoren** [comment]
Idle, using very old version. Please open a new issues if you encounter errors running v2.2

**Comment by sunilthorat** [comment]
Dear brainwalenz and skoren, As suggested I have removed the existing canu-2.2 directory by using "rm canu-2.2 -r" command. Then I resinstalled canu-2.2 from "https://github.com/marbl/canu/releases/tag/v2.2" and then following canu Documentation 1.1 for Pacbio, I performed the below command:

**canu  -p ecoli -d ecoli-pacbio  genomeSize=4.8m  -pacbio-corrected pacbio.fastq**

After sucessfully running the canu command, I got this error again:

-- Starting command on Wed Sep 21 21:17:01 2022 with 1666.972 GB free disk space

    cd trimming
    /home/ibsdshl/anaconda3/libexec/ovStoreBuild \
     -O ./ecoli.ovlStore.BUILDING \
     -G ./ecoli.gkpStore \
     -M 2-8 \
     -L ./1-overlapper/ovljob.files \
     > ./ecoli.ovlStore.err 2>&1

-- Finished on Wed Sep 21 21:17:01 2022 (lickety-split) with 1666.972 GB free disk space
----------------------------------------
ERROR:
ERROR:  Failed with exit code 1.  (rc=256)
ERROR:
================================================================================
Don't panic, but a mostly harmless error occurred and Canu stopped.

Disk space available:  1666.972 GB

Last 50 lines of the relevant log file (trimming/ecoli.ovlStore.err):

Found no overlaps to sort.

Canu release v1.5 failed with:
  failed to create the overlap store

----
Please suggest corrections.

Sunil

**Comment by skoren** [comment]
Your command is still using the canu from the conda install which is the old 1.5 version. You also need to remove all the intermediate outputs you have so far (e.g. `ecoli-pacbio` folder). Provide the full path to where you downloaded the 2.2 release rather than just canu, e.g. `/path/to/version2.2/bin/canu` not `canu`. You're also running the PacBio data as corrected but, unless this is hifi data (in which case you should use -pacbio-hifi), you should specify it as just -pacbio as the tutorial suggests: https://canu.readthedocs.io/en/latest/quick-start.html#assembling-pacbio-clr-or-nanopore-data.
